docs/Architecture/LargeScale.md

```diff
@@ -21,20 +21,47 @@ layout: doc_page
 -->
 ## Designed for Large-scale Computing Systems
 
+#### Multiple Languages
+
+* The DataSketches library is now available in three languages, Java, C++, and Python. A forth language, GoLang, is in development.
```

Comment:
`A fourth language, Go, is in development.`

Typo in forth/fourth
GoLang is the former name of Go https://go.dev/doc/faq#go_or_golang

Reply:
Done.